src/transaction/log_writer.h

```diff
@@ -199,6 +200,10 @@ extern bool logwr_force_shutdown (void);
 extern int logwr_copy_log_file (const char *db_name, const char *log_path, int mode, INT64 start_page_id);
 extern LOG_PHY_PAGEID logwr_to_physical_pageid (LOG_PAGEID logical_pageid);
 extern const char *logwr_log_ha_filestat_to_string (enum LOG_HA_FILESTAT val);
+#if 0
```

Comment:
rather than defining it as #if 0
#if (some featured name) looks better.

Reply:
Thank you for the feedback, I'll reflect it and notify you.

Reply:
I changed from `#ifdef 0` to `#ifdef UNSTABLE_TDE_FOR_REPLICATION_LOG`. Refer to https://github.com/CUBRID/cubrid/pull/2450/commits/fad25194ce61e84e9d47ef8205d0987cbae37467